### PR TITLE
Added register step two

### PR DESCRIPTION
**Pull request by NardusVanDerVyver**

### CODE REVIEW

File: src/containers/register_step2/index.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    color: #FFFFFF;		
+    font-size: 36px;	
+    margin-top: 0px;
+    margin-bottom: 24px;
+    letter-spacing: 2px;	
+    line-height: 49px;	
+    text-align: center;
+}
+
+.ProfileCircle {
```
Comment (jaderabbit) [inline src/containers/register_step2/index.css]: Fix naming inconsistency:

`profile-picture`
`ProfileCircle`

File: src/containers/register_step2/index.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.plus {
+    font-size: 50px; 
+    font-weight: 100;
+    margin: auto;
+    margin-top: -5px;
+}
+
+.btnContainer2 {
```
Comment (jaderabbit) [inline src/containers/register_step2/index.css]: Non descriptive name

File: src/index.css [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+  margin-top: -16px;
 }
 
-.forms{
-  height: 644px;	
-  width: 533px;	
-  border-radius: 12px;	
-  background-color: #FDFDFD;
+ .header{
+  height: 103px;
```
Comment (jaderabbit) [inline src/index.css]: indentation

**Review by jaderabbit — CHANGES_REQUESTED** [review]
So your naming conventions are wrong. 

You components shouldn't have index.css. The index file should be named after the comment. 

Also codmponents should not be defined within each index.js. They should be defined in a insertcomponentnamehere.js

File: src/index.css [hunk 4]
```diff
@@ -1,21 +1,26 @@
...
   background-color: #004853;
   font-family: "Helvetica";
+  margin: 0px;
+  margin-top: -16px;
 }
 
  .header{
   height: 103px;	
-  width: 1440px;	
+  width: 100%;
```
Comment (jaderabbit) [inline src/index.css]: indentation pls

File: src/containers/register_step2/index.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            <p class="step-two">Step Two</p>  
+            <p class="profile-picture">PROFILE PICTURE</p>     
+        
+            <div class="theForm1">
+                <form>
+                    <MuiThemeProvider>
+                        <div class="ProfileCircle">
+                            <button type="button" class="btnProfilePic"><p class="plus">+</p></button>
+                        </div>
+                        <div class="btnContainer2">
```
Comment (jaderabbit) [inline src/containers/register_step2/index.js]: better class name pls

File: src/containers/register_step2/index.css (hunk 1 above)
Comment (jaderabbit) [reply src/containers/register_step2/index.css]: Fix through out

File: src/containers/login/index.js [hunk 6]
```diff
@@ -46,7 +46,7 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = dispatch => bindActionCreators({
     changeToBody: () => push('/body'),
-    changeToRegister: () => push('/register')
+    changeToRegister: () => push('/register_step2')
```
Comment (jaderabbit) [inline src/containers/login/index.js]: `step2` could be improved

File: src/containers/app/index.js [hunk 7]
```diff
@@ -6,6 +6,7 @@ import Body from '../body'
 import Header from '../header'
 import Sidebar from '../sidebar'
 import Chat from '../chat'
+import Register_Step2 from '../register_step2'
```
Comment (jaderabbit) [inline src/containers/app/index.js]: step2 could be improved

File: src/containers/app/index.js [hunk 8]
```diff
@@ -6,17 +6,19 @@ import Body from '../body'
 import Header from '../header'
 import Sidebar from '../sidebar'
 import Chat from '../chat'
+import RegisterStepTwo from '../registerStepTwo'
 
 const App = () => (
     <div>
         <main>
```
Comment (grootendorste) [inline src/containers/app/index.js]: Is main a component you make? Strange I don't see it bweing imported, also usually the components are first letter uppercase.

File: src/containers/app/index.js [hunk 9]
```diff
@@ -6,17 +6,19 @@ import Body from '../body'
...
 import Sidebar from '../sidebar'
 import Chat from '../chat'
+import RegisterStepTwo from '../registerStepTwo'
 
 const App = () => (
     <div>
         <main>
             <Route exact path="/" component={Login} />
-            <Route exact path="/login" component={Login} />
+            <Route exact path="/login.js" component={Login} />
```
Comment (grootendorste) [inline src/containers/app/index.js]: Is it needed to have a ".js" extension, didn't it work previously without it?

Comment (jaderabbit) [inline src/containers/app/index.js]: should it be `.js`?

**Review by grootendorste — CHANGES_REQUESTED** [review]
Just a couple of comments I added.

File: src/containers/app/index.js [hunk 10]
```diff
@@ -6,17 +6,19 @@ import Body from '../body'
...
             <Route exact path="/register" component={Register} />
             <Route exact path="/body" component={Body} />
             <Route exact path="/header" component={Header} />
             <Route exact path="/sidebar" component={Sidebar} />
             <Route exact path="/chat" component={Chat} />
+            <Route exact path="/registerStepTwo" component={RegisterStepTwo} />
```
Comment (jaderabbit) [inline src/containers/app/index.js]: This should really be named better `StepTwo` is a bit vague

**Review by jaderabbit — CHANGES_REQUESTED** [review]
Small things

File: src/containers/app/index.js (hunk 9 above)
Comment (NardusVanDerVyver) [reply src/containers/app/index.js]: No it doesn't need it, so changing it to just "/login" thanks.

File: src/containers/app/index.js (hunk 8 above)
Comment (NardusVanDerVyver) [reply src/containers/app/index.js]: I changed it to be a div, as we don't have a "main" component.

**Review by bsibiya — APPROVED** [review]

**Review by jaderabbit — APPROVED** [review]
:tada:

**Pull request merged**